draft-ietf-quic-load-balancers.md

```diff
@@ -530,9 +527,7 @@ server IDs, including potential future servers.  The server ID will start in the
 second octet of the decrypted connection ID and occupy continuous octets beyond
 that.
 
-The configuration agent selects a zero-padding length. This SHOULD be at least
-four octets to allow detection of non-compliant DCIDs. The server ID and zero-
-padding length MUST sum to no more than 16 octets. They SHOULD sum to no more
+They server ID length MUST be no more than 16 octets and SHOULD sum to no more
 than 12 octets, to provide servers adequate space to encode their own opaque
 data.
 
```

Comment:
There should be some text about the *minimal* length -- long enough to encode all servers, but also long enough to detect errors. Here is probably not the place, it belongs in some common text.

Reply:
In {{routing-algorithms}} it says:

A QUIC-LB configuration MAY significantly over-provision the server ID space
(i.e., provide far more codepoints than there are servers) to increase the
probability that a randomly generated Destination Connection ID is non-
compliant.

I am not particularly convinced this is an important feature, so it's just a MAY. But it's there.

Reply:
OK then. If we have common text that's fine.